website/docs/description/project-description.md

```diff
@@ -1,5 +1,24 @@
 ---
-title: Project description
+title: Project Description
 ---
 
-This is another content
+The advance of the Internet has shattered the old-order educational paradigm where teachers were the almost exclusive owners of knowledge and best practices.
+**In the new-order educational paradigm, information is one click away.**
+The one-click-away availability of information cannot compensate for flaws of the current digital educational content: closed, unstructured, general, static, impractical and non-reproducible.
+
+At Open Education Hub (OpenEdu), we improve the development, delivery and curation of digital educational content by taking in the mindset, best practices and ideas from one of the most successful new-order ventures: the open source collaborative model.
+**We use the open source collaborative model for the development, deployment and curation of digital educational content.**
+The new digital educational content is **public, customizable, dynamic, up-to-date, practical and reproducible**.
+We are developing open educational resources (OER) and we are harnessing the benefits of the open source collaborative model to engage content consumers and content producers.
+In this way, we are facilitating the 5R activities of OER: retain, reuse, revise, remix, redistribute.
```

Comment:
Add [a link](https://nsufl.libguides.com/oer/5rs) that explains the 5R's. Otherwise, they seem a little vague.